silta/silta-prod.yml

```diff
@@ -38,6 +43,17 @@ mariadb:
       # Database storage disk space allocation
       # Request assistance from ops team after changing this on existing deployment.
       size: 5Gi
+    resources:
+      requests:
+        cpu: 250m
+        memory: 500M
+      limits:
+        cpu: 1000m
+        memory: 750M
+
+# Consider enabling memcached service
```

Comment:
Should we even enable this by default? See https://github.com/wunderio/drupal-project/issues/298.

Reply:
cc: @evilfurryone

Reply:
that would be a different issue, this one is about requests and limits for existing servers

Reply:
Ok, thanks.